planet-express/PlanetExpress/src/planet/PlanetExpress.java

```diff
@@ -0,0 +1,110 @@
+package planet;
+
+import java.util.Scanner;
+
+public class PlanetExpress {
+	int thirst;
+	int work;
+	int receipts;
+	int horde;
+	int hunger;
+	boolean stable;
+	
+	PlanetExpress() {
+		thirst = 50;
+		work = 50;
+		receipts = 50;
+		horde = 50;
+		hunger = 50;
+		stable = true;
+		
+	}
+
+	public static void main(String[] args) {
```

Comment:
It would make sense to have a separate file like `Program` that only contained the "runner" part of the program